### PR TITLE
Fix icon anchor scroll

### PR DESCRIPTION
**Pull request by Ricardlol**
fix plugin in this situation:
```html
<a href="#aplicacio" class="btn btn-default">
    <span class="fa fa-suitcase fa-5x __web-inspector-hide-shortcut__" data-original-title="" title="" style=""></span>
</a>
```

### CODE REVIEW

**Comment by yajo** [comment]
Can you explain better on what are you fixing, how to reproduce the error, etc.?

**Comment by Ricardlol** [comment]
Sorry by not answer,
The mistake is that the anchor doesn't work with images, icons, etc, only
work with strings by reproduce the error create a image or icon and add a
anchor
example:
<a href="#example">
    example
</a>

File: website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ odoo.define('website_anchor_smooth_scroll', function(require) {
 
     var smooth_scroll = function(event) {
         event.preventDefault();
-        var anchor_fragment = event.target.hash;
+        var anchor_fragment = event.target.hash || this.hash;
```
Comment (yajo) [inline website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js:14]: Change to `event.currentTarget.hash`

**Review by yajo — CHANGES_REQUESTED** [review]

**Review by tarteo — CHANGES_REQUESTED** [review]
Can you increase the version number in the manifest.

**Pull request closed without merging**